actor/builtin/miner/miner.go

```diff
@@ -146,8 +146,8 @@ type State struct {
 
 	LastUsedSectorID uint64
 
-	ProvingPeriodStart *types.BlockHeight
-	LastPoSt           *types.BlockHeight
+	ProvingPeriodEnd *types.BlockHeight
```

Comment:
Please comment the semantics. I assume that with the name changes comes the semantic meaning "the end of the current proving period" (as opposed to the end of the prior proving period).